cultcargo/casa/mstransform.yml

```diff
@@ -13,6 +13,9 @@ cabs:
             outputvis:
                info: Name of output Measurement Set or Multi-MS
                dtype: str
+            field:
```

Comment:
@tmolteno if you're fixing this cab, could you please pull in all the selection field a-la ``casa.split``:
https://github.com/caracal-pipeline/cult-cargo/blob/dec651a311d461501fdb4b568a10d27a1fb03357/cultcargo/casa/split.yml#L15

Reply:
Done